protos/dcs/mission/v0/mission.proto

```diff
@@ -303,6 +303,20 @@ message StreamEventsResponse {
     dcs.common.v0.Airbase place = 3;
   }
 
+  // Occurs when a chat message is sent on the server
+  message PlayerSentChatEvent {
+    // The player's id in the current server session.
+    uint32 playerSessionId = 1;
```

Comment:
I think including the chat messages into the main event stream is fine and makes perfect sense.

Regarding consistency.
1. Yes our gRPC APIs should be consistent in their field naming whereever possible.
2. We decided when we started the project that we would not be slaves to EDs naming scheme and would use our own API names & our own field names if we thought the ED naming scheme was not very good.  A common case was redundancy in the field names redundancy in the field names.

So consistency in _our_ public API: Yes but sticking to ED naming: No. I probably need to write that down somewhere

As a test case. Here we have `playerSessionId` which is part of the `PlayerSentChatEvent`. We also have an `id` field in the `PlayerConnectEvent` in https://github.com/DCS-gRPC/rust-server/pull/88

I am assuming they mean the same thing here and therefore should be the same. `id` could be argued to be ambiguous (player id? message id? connection id?) however I think `playerSessionId` is also redundant since this is an event _about_ a player. Therefore `sessionId` could be adequate.

However if there are other events that are not specific to being about "A player" then `sessionId` could be ambiguous as well.

I will let others weigh in on which name we should use here.